.github/workflows/main.yml

```diff
@@ -57,6 +57,8 @@ jobs:
       run : | 
         sudo apt-get update
         sudo apt-get install -y libvirt-dev
+    - name: download deps
```

Comment:
Lets make all the steps names Start with Capital letter

Download Deps 
here and other steps too while we are at it.